### PR TITLE
fix(helm-chart-kubeversion-eks): Fixes Helm chart to install on EKS

### PR DESCRIPTION
**Pull request by ironstar94**
**Description**: 
Fixes Helm chart to get it working on EKS Kubernetes clusters. Previously, the install was failing due to kubeVersion being checked too stringently. osm install was returning 

> Error: chart requires kubeVersion: >= 1.18.0 which is incompatible with Kubernetes v1.21.2-eks-0389ca3

as per [Helm's internal issue tracker](https://github.com/helm/helm/issues/3810) >= 1.18.0 would not match 1.18.0-anything, and EKS uses a naming convention that uses dashes after the version, resulting in the above error. This PR resolves the issue, allowing the installation to proceed on EKS clusters

**Affected area**:
| Functional Area            |     |
| -------------------------- | --- |
| New Functionality          | [ ] |
| Documentation              | [ ] |
| Install                    | [X] |
| CLI Tool                   | [ ] |
| Certificate Management     | [ ] |
| Control Plane              | [ ] |
| Ingress                    | [ ] |
| Egress                     | [ ] |
| Networking                 | [ ] |
| Observability              | [ ] |
| SMI Policy                 | [ ] |
| Sidecar Injection          | [ ] |
| Security                   | [ ] |
| Upgrade                    | [ ] |
| Tests                      | [ ] |
| CI System                  | [ ] |
| Demo                       | [ ] |
| Performance                | [ ] |
| Other                      | [ ] |

**Testing done**:

Current release

image

vs

PR 

image

Please answer the following questions with yes/no.

1. Does this change contain code from or inspired by another project? No

2. Is this a breaking change? No

### CODE REVIEW

File: charts/osm/Chart.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ version: 0.9.1
 appVersion: v0.9.1
 
 # This specifies the minimum Kubernetes version OSM is compatible with.
-kubeVersion: ">= 1.19.0"
+kubeVersion: ">= 1.19.0-rc0"
```
Comment (trstringer) [inline charts/osm/Chart.yaml:24]: Reading through https://github.com/helm/helm/issues/3810 it appears as though the pre-release string can be anything. Using `rc0` is very specific and could leave other readers wondering if there is some logic that checks for common pre-release strings, which it appears there is not. My preference would be to use `-0` as it is a common logical base.

**Review by trstringer — CHANGES_REQUESTED** [review]

File: charts/osm/Chart.yaml (hunk 1 above)
Comment (ironstar94) [reply charts/osm/Chart.yaml]: Makes sense. I've updated it to `-0`

**Review by trstringer — APPROVED** [review]
Thank you for the contribution!

**Review by shashankram — APPROVED** [review]

**Comment by ironstar94** [comment]
@trstringer Any chance this could be included in a backport for 0.9x release?

**Comment by shashankram** [comment]
> @trstringer Any chance this could be included in a backport for 0.9x release?

@ironstar94 we will include this if we decide to cut the v0.9.2 patch release. Till now, there wasn't a need to cut a patch release on v0.9, but if this becomes a blocker for your use case we could cut backport this and cut a release.

**Comment by ironstar94** [comment]
> > @trstringer Any chance this could be included in a backport for 0.9x release?
> 
> @ironstar94 we will include this if we decide to cut the v0.9.2 patch release. Till now, there wasn't a need to cut a patch release on v0.9, but if this becomes a blocker for your use case we could backport this and cut a release.

I've recompiled OSM cli locally, but I'm not the only user, and we would like to make our reproduce the environment without relying on internal hacks. Cutting a 0.9.2 patch release would be greatly appreciated.

**Comment by shashankram** [comment]
> > > @trstringer Any chance this could be included in a backport for 0.9x release?
> > 
> > 
> > @ironstar94 we will include this if we decide to cut the v0.9.2 patch release. Till now, there wasn't a need to cut a patch release on v0.9, but if this becomes a blocker for your use case we could backport this and cut a release.
> 
> I've recompiled OSM cli locally, but I'm not the only user, and we would like to make our reproduce the environment without relying on internal hacks. Cutting a 0.9.2 patch release would be greatly appreciated.

@ironstar94 we will cut v0.9.2 mid next week. I'll backport this, thanks!

**Pull request merged**